src/test/java/com/google/api/codegen/protoannotations/testdata/go_library.baseline

```diff
@@ -2205,6 +2241,18 @@ func (s *mockLibraryServer) ListShelves(ctx context.Context, req *librarypb.List
     return s.resps[0].(*librarypb.ListShelvesResponse), nil
 }
 
+func (s *mockLibraryServer) ListAggregatedShelves(ctx context.Context, req *librarypb.AggregatedListShelvesRequest) (*librarypb.ListShelvesAggregatedResponse, error) {
```

Comment:
I would expect that (if we were still using the monolith for Go), this would have a similar structure to `func (c *LibClient) ListShelves(...){...}`, dealing with `NextPageToken`, etc. Is that right? Is the only reason that's not the case simply that real Go generation is happening in the µgenerator? (That's totally fine; I just want to make sure I'm not missing anything.)

Reply:
Line 247 in PageStreamingConfig.java explicitly asks to use the new logic for Java only. It is important, because it will start breaking other languages in monolith (most importantly PHP). I.e. only java gax supports ma-typed pagination, that is why only Java gapic generation does it.

Reply:
Ah, got it. So this is a monolith-only problem. ;-)